src/components/Rating.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { Children } from 'react'
+
+function Rating(props) {
+    const {children} = props;
+    function stars (children) {
+        if (children >= 4.5) return '★★★★★';
+        else if (children >= 3.5) return '★★★★☆';
+        else if (children >=2.5) return '★★★☆☆';
+        else if (children >=1.5) return '★★☆☆☆';
+        else if (children >=0.5) return '★☆☆☆☆';
+        else if (children === 0) return '☆☆☆☆☆';
```

Comment:
Here for your logic to work, you would need just two =, like:

` else if (children == 0) return '☆☆☆☆☆';`